Content.Server/Forensics/Systems/ForensicPadSystem.cs

```diff
@@ -80,8 +80,10 @@ private void OnAfterInteract(EntityUid uid, ForensicPadComponent component, Afte
                 return;
             }
 
+            // Begin DeltaV changes - add Fiberprint to display
             if (TryComp<FiberComponent>(args.Target, out var fiber))
-                StartScan(uid, args.User, args.Target.Value, component, string.IsNullOrEmpty(fiber.FiberColor) ? Loc.GetString("forensic-fibers", ("material", fiber.FiberMaterial)) : Loc.GetString("forensic-fibers-colored", ("color", fiber.FiberColor), ("material", fiber.FiberMaterial)));
+                StartScan(uid, args.User, args.Target.Value, component, string.IsNullOrEmpty(fiber.FiberColor) ? Loc.GetString("forensic-pad-fibers", ("material", fiber.FiberMaterial), ("fiberprint", fiber.Fiberprint ?? string.Empty)) : Loc.GetString("forensic-pad-fibers-colored", ("color", fiber.FiberColor), ("material", fiber.FiberMaterial), ("fiberprint", fiber.Fiberprint ?? string.Empty)));
+            // End DeltaV changes
```

Comment:
It might be for the best, just for readability sake. Here's my suggestion, though you are free to organize it however you'd like.

```suggestion
            // Begin DeltaV changes - add Fiberprint to display
            if (TryComp<FiberComponent>(args.Target, out var fiber))
            {
                var text = Loc.GetString("forensic-pad-fibers", ("material", fiber.FiberMaterial), ("fiberprint", fiber.Fiberprint ?? string.Empty));

                if (!string.IsNullOrEmpty(fiber.FiberColor)) // If there are colored fibers, use that locale instead
                    text = Loc.GetString("forensic-pad-fibers-colored", ("color", fiber.FiberColor), ("material", fiber.FiberMaterial), ("fiberprint", fiber.Fiberprint ?? string.Empty));

                StartScan(uid, args.User, args.Target.Value, component, text);
            }
            // End DeltaV changes
```

Reply:
And the line was pretty gross before you changed it, so don't think I meant what you did specifically was bad hah,

Reply:
Yeah I get it! Again, I preferred to just let it be at first, haven't touched the C# part of the code too much yet.
I'll accept your suggestion for sure tho!
...just gotta do it manually github is being annoying about it